app/helpers/media_objects_helper.rb

```diff
@@ -131,10 +131,11 @@ def hide_sections? sections
      def structure_html section, index, show_progress
        current = is_current_section? section
        progress_div = show_progress ? '<div class="status-detail alert" style="display: none"></div>' : ''
-       playlist_btn = current_ability.can?(:create, Playlist) ? '<button type="button" title="Add section to playlist" aria-label="Add section to playlist" class="structure_add_to_playlist outline_on btn btn-primary" data-scope="master_file" data-masterfile-id="#{section.id}"></button>' : ''
+
+       playlist_btn = current_ability.can?(:create, Playlist) ? "<button type=\"button\" title=\"Add section to playlist\" aria-label=\"Add section to playlist\" class=\"structure_add_to_playlist outline_on btn btn-primary\" data-scope=\"master_file\" data-masterfile-id=\"#{section.id}\"></button>" : ''
 
        headeropen = <<EOF
-    <div class="panel-heading" role="tab" id="heading#{index}">
+    <div class="panel-heading" role="tab" id="heading#{index}" data-media-object-id="#{section.isPartOf}" data-section-id="#{section.id}">
```

Comment:
I think`isPartOf` works here, but we usually use `media_object_id` instead.  Do you mind switching it?

Reply:
Yea, totally.  I didn't know that was available.  I just did a `<% byebug %> to see what I could grab.  One of these days I need a Rails tutorial on how to view full object state or something.  Thanks for looking at this...